src/HealthChecks/Abstractions/src/HealthCheckContext.cs

```diff
@@ -8,6 +8,6 @@ public sealed class HealthCheckContext
         /// <summary>
         /// Gets or sets the <see cref="HealthCheckRegistration"/> of the currently executing <see cref="IHealthCheck"/>.
         /// </summary>
-        public HealthCheckRegistration Registration { get; set; }
+        public HealthCheckRegistration Registration { get; set; } = default!;
```

Comment:
Was this only nullable for testing? Should add comment why it has been forced to non-nullable

Reply:
Yeah, the calling code assumes this is non-null.

Reply:
Done